src/main/java/com/google/sps/data/User.java

```diff
@@ -19,14 +19,28 @@
 /** Class representing a user. */
 public class User {
 
-    // Contains the entity id from Datastore.
-    private long id;
+    // String representation of the key from Datastore.
```

Comment:
In this case, I believe you're using a "key name string" and not an id - it would be change the name to more accurately reflect the subject.

https://cloud.google.com/datastore/docs/concepts/entities